VERSIONS.md

```diff
@@ -1,107 +1,108 @@
-| 4.6.0   | 3.14.1      | 4.6.1           | 2.0.1                |
-| 4.5.3   | 3.14.1      | 4.6.1           | 2.0.1                |
-| 4.5.2   | 3.14.1      | 4.6.1           | 2.0.1                |
-| 4.5.1   | 3.13.1      | 4.6.0           | 1.11.1               |
-| 4.5.0   | 3.13.1      | 4.6.0           | 1.11.1               |
-| 4.4.1   | 3.10.0      | 4.4.0           | 1.10.0               |
-| 4.4.0   | 3.10.0      | 4.4.0           | 1.10.0               |
-| 4.3.3   | 3.12.8      | 4.3.3           | 1.9.3                |
-| 4.3.2   | 3.12.3      | 4.3.1           | 1.9.1                |
-| 4.3.1   | 3.12.3      | 4.3.1           | 1.9.1                |
-| 4.3.0   | 3.12.2      | 4.3.0           | 1.8.0                |
-| 4.2.0   | 3.11.1      | 4.2.1           | 1.7.1                |
-| 4.1.4   | 3.11.1      | 4.2.1           | 1.7.1                |
-| 4.1.3   | 3.11.1      | 4.2.1           | 1.7.1 (Android 1.7.0)|
-| 4.1.2   | 3.11.1      | 4.0.5           | 1.6.3                |
-| 4.1.1   | 3.10.7      | 4.0.5           | 1.6.2                |
-| 4.1.0   | 3.10.6      | 4.0.4           | 1.6.1                |
-| 4.0.1   | 3.9.2       | 4.0.1           | 1.5.0                |
-| 4.0.0   | 3.9.2       | 4.0.1           | 1.5.0                |
-| 3.4.3   | 3.7.5       | 3.5.2           | 1.4.5                |
-| 3.4.2   | 3.7.2       | 3.5.2           | 1.4.4                |
-| 3.4.1   | 3.7.2       | 3.5.0           | 1.4.3                |
-| 3.4.0   | 3.7.1       | 3.5.0           | 1.4.2                |
-| 3.3.3   | 3.5.1       | 3.3.0           | 1.3.1                |
-| 3.3.2   | 3.4.0       | 3.2.0           | 1.2.0                |
-| 3.3.1   | 3.4.0       | 3.2.0           | 1.2.0                |
-| 3.3.0   | 3.4.0       | 3.2.0           | 1.2.0                |
-| 3.2.0   | 3.2.1       | 3.1.0           | 1.0.10               |
-| 3.1.0   | 3.2.1       | 3.1.0           | 1.0.9                |
-| 3.0.6   | 3.0.1       | 3.0.4           | 1.0.5                |
-| 3.0.5   | 3.0.1       | 3.0.4           | 1.0.5                |
-| 3.0.4   | 3.0.1       | 3.0.4           | 1.0.5                |
-| 3.0.3   | 3.0.1       | 3.0.4           | 1.0.4                |
-| 3.0.2   | 3.0.0       | 3.0.3           | 1.0.3                |
-| 3.0.1   | 3.0.0       | 3.0.3           | 1.0.1                |
-| 3.0.0   | 3.0.0       | 3.0.2           | 1.0.1                |
-| 2.4.1   | 2.6.0       | 2.4.0           | 0.1.4                |
-| 2.4.0   | 2.6.0       | 2.4.0           | 0.1.3                |
-| 2.3.4   | 2.5.0       | 2.3.1           | 0.1.2                |
-| 2.3.3   | 2.5.0       | 2.3.1           | 0.1.2                |
-| 2.3.2   | 2.5.0       | 2.3.0           | 0.1.2                |
-| 2.3.1   | 2.5.0       | 2.3.0           | 0.1.1                |
-| 2.3.0   | 2.4.0       | 2.3.0           | N/A                  |
+| Version       | iOS version | Android version | Common files version  |
```

Comment:
Oh I guess this was formatted manually? I think it would be good to automate this formatting, but don't think it's a priority. We can go with this for now though!